### ISSUE TITLE
Error in running experiments

### ISSUE THREAD

**Issue by yb6599** [Jacob-Stevens-Haas/gen-experiments#12]
Hey Jake,

I am getting errors while running the experiments. The error is as follows
`Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\UW\Research\gen-experiments\src\gen_experiments\__main__.py", line 39, in <module>
    params = gen_experiments.lookup_params(args.param)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\UW\Research\gen-experiments\src\gen_experiments\__init__.py", line 102, in lookup_params
    resolved_params.append(Parameter(p_id, p_name, vals, modules))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Parameter.__init__() takes 4 positional arguments but 5 were given`

I am trying to find out why this error occurs. You got any ideas?

**Comment by Jacob-Stevens-Haas** [comment]
You're on the new version of `mitosis`.  I'll talk more about this tomorrow if you're able to meet, otherwise I'll add some release notes to version 0.3.0 of mitosis.

tl;dr: There's some big changes to mitosis driven by problems I've had running complex experiments.  Because there's other people using `mitosis` now, I'm going to get better about actually releasing versions though.

what version of mitosis do you have, and which commit of `gen-experiments` are you running the experiments on?

**Comment by yb6599** [comment]
I will be able to meet, yes.
Mitosis version: `0.3.0b2.dev12+gc0e4b22`
gen-experiments latest commit: `commit 551ac0242479978833bed105433764fdea99ca99`
